lib/spack/spack/build_environment.py

```diff
@@ -168,8 +169,16 @@ def __call__(self, *args, **kwargs):
                     fail_on_error=False
                 )
                 self.supports_sync = self.returncode == 0
+                super(MakeExecutable, self).__call__(
+                    "--no-silent",
```

Comment:
What's the use of `--no-silent` really? I just tried it and it doesn't overrule a `.SILENT:` target for instance.

Reply:
Fair question, it can't override everything, but it ensures that `-s` or some of the silencing methods don't silence it.  That combined with the `V` and `VERBOSE` env vars seems to help more often than not, it's not completely required, but I don't know a better way to accomplish it.